atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -106,18 +101,24 @@ public void shutdown() {
 
     private static final SafeLogger log = SafeLoggerFactory.get(CassandraClientPoolImpl.class);
 
+    private static final String EMPTY_CLIENT_POOL_MESSAGE =
```

Comment:
What I _think_ you were trying to do is make this public and use it in the test where we you do LoggableExceptionThrownBy. However, that's not the case (it's just used in the one precondition). As I mentioned in the MTD PR, we _generally_ inline the message both here and the test. You could _feasibly_ get away with just testing that the message contains `No servers were successfully added to the pool`, since the test is just to assert that it's our exception that we're throwin